### DISCUSSION TITLE
Program FPGA from Mac via OpenOCD?

### DISCUSSION THREAD

**Discussion by NilsGraf** [ichi4096/vivado-on-silicon-mac#44] [Ideas]
Have you considered using [OpenOCD](https://openocd.org/) on Mac to program the bitstream file into the FPGA?  See [these instructions](https://github.com/byu-cpe/BYU-Computing-Tutorials/wiki/Program-7-Series-FPGA-from-a-Mac-or-Linux-Without-Xilinx) on how to do this for Series 7 FPGAs.  Thanks!

**Comment by ichi4096** [comment]
You can do that, but since this relies on using the terminal manually on the macOS side, I don't see what I could add to my scripts.
